circuit/account/src/compute_key/helpers/to_fields.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2019-2023 Aleo Systems Inc.
+// This file is part of the snarkVM library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::*;
+
+impl<A: Aleo> ToFields for ComputeKey<A> {
+    type Field = Field<A>;
+
+    /// Casts a string into a list of base fields.
+    fn to_fields(&self) -> Vec<Self::Field> {
+        vec![self.pk_sig().to_x_coordinate(), self.pr_sig().to_x_coordinate()]
```

Comment:
Use `.to_field()` to be matching syntax.

Add a test that `console` and `circuit` matches.